src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+a kind of ether of determinateness that connects all objects together but does
+not affect them beyond connecting them.
+
+### Communication Between Objects
+
+Hegel then embarks on a remark that helpfully unpacks what he has in mind with
+the logical determination of `communication`. He writes:
+
+> _Spiritual communication_, which moreover takes place in that element which is
+> the universal in the form of universality, is explicitly an _ideal_ relation
+> in which _a determinateness continues_ itself from one person into another
+> unimpaired, and universalizes itself without any alteration whatever-as a
+> scent freely spreads in the unresisting atmosphere. But even in communication
+> between material objects, their determinateness _spreads_, so to speak, in a
+> similarly ideal manner; personality is an infinitely more intense
+> _impenetrability_ [*Harte*] than objects possess (Hegel 1991, 716). The above
+> paragraph provides a series of concrete instances of `communication`.
+> Crucially, the examples given are all of the kind of communication that occurs
+> between objects; Hegel makes the point to distinguish the kind of
+> communication between objects to something like personality, which “is an
+> infinitely more intense _impenetrability_ [*Harte*] than objects possess”
+> (Hegel 1991, 716). This is an excellent opportunity to look closely at Hegel’s
+> examples as they provide a window into how he thinks about the real-world
+> instances of the logical development. If we spend a bit of time unpacking
+> Hegel’s examples and scrutinising their validity, we will get a clearer
+> understanding of how he thinks of ontology.
+
+Beginning with Hegel’s first example, that of a scent diffusing in an atmosphere
+where there is no restriction on the diffusion of the scent. Why does Hegel
+think that this is a helpful example of how `communication` is exemplified in
+everyday life? At first glance, it makes sense. The way a scent freely diffuses
+itself around an atmosphere so that the whole atmosphere smells of the scent is
+reminiscent of the identical determinateness of the `mechanical object` that
+leads to it becoming a universal. However, how _exactly_ are we supposed to
+understand this? Is the mechanical process at work between the scented molecules
+and the non-scented molecules, or is it just between the scented molecules? To
+pose this question is, I think, to overstep the conceptual restrictions of
+Mechanism and mechanistic thought. The distinction between a scented molecule
+and a non-scented molecule does not obtain within a mechanistic framework
+because, as `mechanical objects`, they do not have different determinations
+(_i.e._ one is scented and the other is non-scented). Quite simply, scented
+molecules and non-scented molecules have an identical determinateness.
+
+### Interlude: How does the Science of Logic relate to External Reality?
+
+A further philosophical conundrum poses itself. Above, I said that the
```

Comment:
Who is "I"? If you mean to add a section in your own voice, if so, add your last name to the heading in parantheses.

`### Interlude: How does the Science of Logic relate to External Reality? (Rokni)` 

This indicates that the section is flagged as a personal contribution. 🙂

Reply:
Tight idea. Have implemented.